application/models/UserDatabase.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+class UserDatabase {
+
+  protected $connection;
+
+  public function __construct() {
+    $this->connection = new mysqli(SERVER, USER, PASS, DBNAME);
+    // Check connection
+    if ($this->connection->connect_error) {
+      die("Connection failed: " . $conn->connect_error);
+    }
+  }
+
+  public function insertUserInfo($name, $email, $dob, $gender, $pass) {
+    $sql = "insert into user(name, email, dob, gender, passcode) 
+      values ('$name', '$email', '$dob', '$gender', '$pass')";
+    if ($this->connection->query($sql)) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  public function emailExist($email) {
+    $sql = "select email from user";
```

Comment:
Can't we use $email in this query instead of fetching all emails in the database and then looping over them?